File: java/oop/CollectionApp/src/com/techlabs/collection/LineItem.java
package com.techlabs.collection;

public class LineItem {
	private int id;
	private int quantity;
	private String name;
	private double price;

	public LineItem(String name, int id, double price,int quantity) {
		this.name = name;
		this.id = id;
		this.price = price;
		this.quantity=quantity;
	}
	public int getId(){
		return this.id;
	}
	public String getItemName(){
		return this.name;
	}
	public int getQuantity(){
		return this.quantity;
	}
	public double getPrice(){
		return this.price;
	}
	public double totalCost() {
		double totalCost;
		totalCost = this.quantity * price;
		return totalCost;
	}
}

File: dotNet/csharp/CollectionApp/CollectionApp/LineItem.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CollectionApp
{
    class LineItem
    {
        private int _id;
        private int _quantity;
        private string _name;
        private double _price;

        public LineItem(string name,int id,double price, int quantity)
        {
            this._name = name;
            this._id = id;
            this._quantity = quantity;
            this._price = price;
        }

        public string Name
        {
            get
            {
                return _name;
            }
        }

        public int Id
        {
            get
            {
                return _id;
            }
        }

        public int Quantity
        {
            get
            {
                return _quantity;
            }
        }

        public double Price
        {
            get
            {
                return _price;
            }
        }

        public double TotalCost()
        {
            double totalCost;
            totalCost= _quantity * _price;
            return totalCost;
        }
    }
}
